public/config/configuration.json

```diff
@@ -14,12 +14,15 @@
   "NAICS_API_VERSION": "/api/v2",
   "PAY_API_URL": "https://pay-api-dev.apps.silver.devops.gov.bc.ca",
   "PAY_API_VERSION": "/api/v1",
-  "SENTRY_ENABLE": "false",
   "SENTRY_DSN": "",
+  "SENTRY_ENABLE": "false",
   "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi",
   "STATUS_API_URL": "https://status-api-dev.apps.silver.devops.gov.bc.ca",
   "STATUS_API_VERSION": "/api/v1",
+  "REGISTRY_HOME_URL": "https://yfthig-dev.web.app/",
   "REGISTRIES_SEARCH_API_URL": "https://bcregistry-dev.apigee.net/registry-search",
   "REGISTRIES_SEARCH_API_VERSION": "/api/v1",
-  "BUSINESS_API_KEY": "KHGuiTxG52ejzFZi7wu4QyD8GdVGEMAz"
+  "WEBCHAT_REASON": "",
+  "WEBCHAT_STATUS_URL": "",
+  "WEBCHAT_URL": ""
```

Comment:
I added these keys to 1Password (vault: entity, application: business-create-ui). I copied the Namerequest keys for Dev only. All 3 environments will need new keys after we talk with Maximus (ticket https://app.zenhub.com/workspace/o/bcgov/entity/issues/12985).

For local testing, I inserted the Dev keys here. (Works fine.) But we don't want any keys here by default.

Reply:
@pwei1018 FYI ^^^